flaml/autogen/agentchat/responsive_agent.py

```diff
@@ -891,6 +897,11 @@ def execute_code_blocks(self, code_blocks):
                 )
                 # raise NotImplementedError
             self._code_execution_config["use_docker"] = image
+
+            if self.auto_reply_token_limit > 0 and count_token(logs_all + "\n" + logs) > self.auto_reply_token_limit:
+                logs_all += "\n" + "Error: The output exceeds the length limit and is truncated."
```

Comment:
Add "If full output is desired, consider writing the output to a file."? @afourney

Reply:
@sonichi Yes, the full output is often needed. We'll want to write it to a file, and return a link or path to the file.